src/Nav.php

```diff
@@ -24,13 +24,7 @@ final class Nav extends Widget
 
     protected function run(): string
     {
-        $html = $this->renderItems();
-
-        if ($this->dropdown) {
-            $html .= Html::endTag('div');
-        }
-
-        return $html;
+        return $this->renderItems();
```

Comment:
There's no need for separate `renderItems()` method, the code could be moved to `run()`. The change itself is alright.

Reply:
Move.